Projects/App/Sources/Screens/ChatScreen/ChatState.swift

```diff
@@ -164,6 +165,14 @@ class ChatState {
             })
             .valueSignal
             .compactMap(on: .concurrentBackground) { data -> [OctopusGraphQL.MessageFragment]? in
+                UserDefaults.standard.set(
```

Comment:
You know what?
Lets move this and use new property in homeState where we keep this date.
Also action should be added to set this property.